docker/Dockerfile

```diff
@@ -86,9 +86,9 @@ RUN apt-get update && \
 WORKDIR /opt/perseo-fe
 
 RUN \
-	# Ensure that Git is installed prior to running npm install
+	# Ensure that Git and curl are installed prior to running npm install
 	apt-get update && \
-	apt-get install -y git && \
+	apt-get install -y git curl && \
```

Comment:
Why curl is needed? Some installation step is using it?

Reply:
curl is used always by default HEALTHCHECK. If you build locally image without https://github.com/telefonicaid/perseo-fe/blob/master/docker/Dockerfile#L60-L84 block then docker fails

Reply:
NTC